Cargo.lock

```diff
@@ -2473,6 +2714,36 @@ dependencies = [
  "windows-sys 0.52.0",
 ]
 
+[[package]]
+name = "notify"
+version = "6.1.1"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "6205bd8bb1e454ad2e27422015fb5e4f2bcc7e08fa8f27058670d208324a4d2d"
+dependencies = [
+ "bitflags 2.5.0",
+ "crossbeam-channel",
+ "filetime",
+ "fsevent-sys",
+ "inotify",
+ "kqueue",
+ "libc",
+ "log",
+ "mio",
+ "walkdir",
+ "windows-sys 0.48.0",
+]
+
```

Comment:
Ah, this is the issue—this is a new dependency *here*. My spelunking earlier saw that it was around in `rust-lang/mdbook` for quite some time. I will see if I can figure out why it was not already a dependency here given this *did* already have a dependency on `mdbook`, which has had the `notify` dependency for some timem.

Reply:
Ah, the problem is that the preprocessors depend on `mdBook` but do not specify `default-features = false`. I will fix that and then update this PR with the correct new commit from `rust-lang/book`.

Reply:
Fixed in 81b2f961fd4; this *should* be good to go now. Let’s see what CI says!

Reply:
✅ for tidy! Resolving this.